src/kindhandlers/common/customMatchers.ts

```diff
@@ -4,7 +4,7 @@ import {K8sResource} from '@models/k8sresource';
  * Matcher that ensures the source and target namespace are the same
  */
 
-export function implicitNamespaceMatcher(sourceResource: K8sResource, targetResource: K8sResource, value: string) {
+export function implicitNamespaceMatcher(sourceResource: K8sResource, targetResource: K8sResource, _: string) {
```

Comment:
Should _ still be here since it's not used?

Reply:
yes my bad - mixing up languages - changing back to value

Reply:
@andreiv1992 the value is included in the contract for matcher methods - its just that this matcher doesn't use it